gatsby-node.js

```diff
@@ -186,24 +184,24 @@ function createWorks({ boundActionCreators, graphql }) {
       .map(d => d.works.map(p => Object.assign({}, p, { designer: d })))
       .reduce((arr, p) => arr.concat(p), [])
 
-    const classifiedTags = getAllTags(designers)
-    // create page for each category and tag
-    classifiedTags.forEach(({ category, tags }) => {
-      createPage({
-        path: categoryLink(category),
-        component: categoryTemplate,
-        context: { category },
-      })
+    const tags = getAllTags(designers)
 
-      tags.forEach(tag => {
-        createPage({
-          path: workTagLink(tag),
-          component: worksByTagTemplate,
-          context: { category, tag },
-        })
-      })
+    // root will be first tag (?)
```

Comment:
We had something like this in an earlier version of the site and it was sort of confusing breadcrumbs-wise. I think it would probably make more sense to have a text-based list of the available tags here, similar to how it will be for designers and projects (as you mentioned on slack). I will make a component that should be droppable in here when I work on that refactor.

Reply:
Cool, sounds good